SparseGrids/tasgridUnitTests.cpp

```diff
@@ -786,4 +800,69 @@ std::vector<std::function<void(void)>> GridUnitTester::getRuntimeErrorCalls() co
     };
 }
 
+bool GridUnitTester::testLAPACKInterface() {
+
+    bool all_matched = true;
+    #ifdef Tasmanian_ENABLE_BLAS
+    // Initialize.
+    const int N = 100;
+    double* exact_eigs = new double[N];
```

Comment:
NO!

Use RAII and `std::vector`. The clauses for `new` and `delete` should only ever be used in very VERY special circumstances. If they can be avoided, then they should always be avoided.

Watch this:
https://www.youtube.com/watch?v=7Qgd9B1KuMQ

Reply:
Done.